**Windows** [Build 3930](http://brontes.lab.eng.brq.redhat.com/viewLog.html?buildId=81458&buildTypeId=WF_PullRequest_Windows) outcome was **FAILURE** using a merge of b8d167b9718b3df9795e7bcc5579e94e40247b02
Summary: Tests failed: 29 (29 new), passed: 3261, ignored: 252 Build time: 01:00:27
### Failed tests
```
org.jboss.as.test.smoke.messaging.client.jms.JmsClientTestCase.testSendAndReceive: org.jboss.naming.remote.protocol.NamingIOException: Failed to lookup
	at org.jboss.naming.remote.client.ClientUtil.namingException(ClientUtil.java:49)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.execute(Protocol.java:104)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1.lookup(RemoteNamingStoreV1.java:95)
	at org.jboss.naming.remote.client.HaRemoteNamingStore$1.operation(HaRemoteNamingStore.java:276)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.namingOperation(HaRemoteNamingStore.java:137)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.lookup(HaRemoteNamingStore.java:272)
	at org.jboss.naming.remote.client.RemoteContext.lookupInternal(RemoteContext.java:104)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:93)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:146)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at org.jboss.as.test.smoke.messaging.client.jms.JmsClientTestCase.doSendAndReceive(JmsClientTestCase.java:103)
	at org.jboss.as.test.smoke.messaging.client.jms.JmsClientTestCase.testSendAndReceive(JmsClientTestCase.java:97)
Caused by: java.io.EOFException: Read past end of file
	at org.jboss.marshalling.SimpleDataInput.eofOnRead(SimpleDataInput.java:151)
	at org.jboss.marshalling.SimpleDataInput.readUnsignedByteDirect(SimpleDataInput.java:294)
	at org.jboss.marshalling.SimpleDataInput.readIntDirect(SimpleDataInput.java:343)
	at org.jboss.marshalling.SimpleDataInput.readLongDirect(SimpleDataInput.java:325)
	at org.jboss.marshalling.SimpleDataInput.readLong(SimpleDataInput.java:320)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadClassDescriptor(RiverUnmarshaller.java:1000)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadNewObject(RiverUnmarshaller.java:1255)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:276)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:209)
	at org.jboss.marshalling.AbstractObjectInput.readObject(AbstractObjectInput.java:41)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:156)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.BaseProtocolCommand.readResult(BaseProtocolCommand.java:59)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.handleClientMessage(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1$MessageReceiver$1.run(RemoteNamingStoreV1.java:232)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
------- Stdout: -------
15:47:44,909 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-4) WFLYSRV0028: Stopped deployment messaging-example.jar (runtime-name: messaging-example.jar) in 5ms
15:47:44,918 INFO  [org.jboss.as.repository] (management-handler-thread - 1) WFLYDR0002: Content removed from location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\smoke\target\jbossas\standalone\data\content\34\09c9a5badfe239b22ead87f9718b03dc16310d\content
15:47:44,918 INFO  [org.jboss.as.server] (management-handler-thread - 1) WFLYSRV0009: Undeployed "messaging-example.jar" (runtime-name: "messaging-example.jar")
15:47:44,926 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 16) AMQ221003: trying to deploy queue jms.queue.createdTestQueue
15:47:44,936 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "createdTestQueue")
    ],
    "entries" => ["java:jboss/exported/createdTestQueue"]
}
15:47:45,031 ERROR [org.jboss.as.naming] (pool-1-thread-3) WFLYNAM0008: Unexpected internal error: java.lang.NoClassDefFoundError: com/google/common/net/InetAddresses
	at org.apache.activemq.artemis.utils.IPV6Util.encloseHost(IPV6Util.java:39)
	at org.apache.activemq.artemis.uri.TCPServerLocatorSchema.escapeIPv6Host(TCPServerLocatorSchema.java:82)
	at org.apache.activemq.artemis.uri.TCPServerLocatorSchema.getURI(TCPServerLocatorSchema.java:75)
	at org.apache.activemq.artemis.uri.TCPSchema.internalNewURI(TCPSchema.java:66)


org.jboss.as.test.integration.ee.remotelookup.LookupTestCase.testClientRemoteLookup: org.jboss.naming.remote.protocol.NamingIOException: Failed to lookup
	at org.jboss.naming.remote.client.ClientUtil.namingException(ClientUtil.java:49)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.execute(Protocol.java:104)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1.lookup(RemoteNamingStoreV1.java:95)
	at org.jboss.naming.remote.client.HaRemoteNamingStore$1.operation(HaRemoteNamingStore.java:276)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.namingOperation(HaRemoteNamingStore.java:137)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.lookup(HaRemoteNamingStore.java:272)
	at org.jboss.naming.remote.client.RemoteContext.lookupInternal(RemoteContext.java:104)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:93)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:146)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at org.jboss.as.test.integration.ee.remotelookup.LookupTestCase.lookupConnectionFactory(LookupTestCase.java:79)
	at org.jboss.as.test.integration.ee.remotelookup.LookupTestCase.testClientRemoteLookup(LookupTestCase.java:75)
Caused by: java.io.EOFException: Read past end of file
	at org.jboss.marshalling.SimpleDataInput.eofOnRead(SimpleDataInput.java:151)
	at org.jboss.marshalling.SimpleDataInput.readUnsignedByteDirect(SimpleDataInput.java:294)
	at org.jboss.marshalling.SimpleDataInput.readIntDirect(SimpleDataInput.java:343)
	at org.jboss.marshalling.SimpleDataInput.readLongDirect(SimpleDataInput.java:325)
	at org.jboss.marshalling.SimpleDataInput.readLong(SimpleDataInput.java:320)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadClassDescriptor(RiverUnmarshaller.java:1000)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadNewObject(RiverUnmarshaller.java:1255)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:276)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:209)
	at org.jboss.marshalling.AbstractObjectInput.readObject(AbstractObjectInput.java:41)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:156)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.BaseProtocolCommand.readResult(BaseProtocolCommand.java:59)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.handleClientMessage(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1$MessageReceiver$1.run(RemoteNamingStoreV1.java:232)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
------- Stdout: -------
15:48:56,208 ERROR [org.jboss.as.naming] (pool-1-thread-1) WFLYNAM0008: Unexpected internal error: java.lang.NoClassDefFoundError: com/google/common/net/InetAddresses
	at org.apache.activemq.artemis.utils.IPV6Util.encloseHost(IPV6Util.java:39)
	at org.apache.activemq.artemis.uri.TCPServerLocatorSchema.escapeIPv6Host(TCPServerLocatorSchema.java:82)
	at org.apache.activemq.artemis.uri.TCPServerLocatorSchema.getURI(TCPServerLocatorSchema.java:75)
	at org.apache.activemq.artemis.uri.TCPSchema.internalNewURI(TCPSchema.java:66)
	at org.apache.activemq.artemis.uri.TCPSchema.internalNewURI(TCPSchema.java:31)
	at org.apache.activemq.artemis.utils.uri.URISchema.newURI(URISchema.java:45)
	at org.apache.activemq.artemis.utils.uri.URIFactory.createSchema(URIFactory.java:78)
	at org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory.toURI(ActiveMQConnectionFactory.java:119)
	at org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory.writeExternal(ActiveMQConnectionFactory.java:83)
	at org.jboss.marshalling.river.RiverMarshaller.doWriteObject(RiverMarshaller.java:842)
	at org.jboss.marshalling.AbstractObjectOutput.writeObject(AbstractObjectOutput.java:58)
	at org.jboss.marshalling.AbstractMarshaller.writeObject(AbstractMarshaller.java:111)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$2.write(Protocol.java:138)
	at org.jboss.naming.remote.protocol.v1.WriteUtil.write(WriteUtil.java:61)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.handleServerMessage(Protocol.java:128)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingServerV1$MessageReciever$1.run(RemoteNamingServerV1.java:73)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)


org.jboss.as.test.integration.ejb.mdb.containerstart.SendMessagesTestCase.testShutdown: org.jboss.naming.remote.protocol.NamingIOException: Failed to lookup
	at org.jboss.naming.remote.client.ClientUtil.namingException(ClientUtil.java:49)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.execute(Protocol.java:104)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1.lookup(RemoteNamingStoreV1.java:95)
	at org.jboss.naming.remote.client.HaRemoteNamingStore$1.operation(HaRemoteNamingStore.java:276)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.namingOperation(HaRemoteNamingStore.java:137)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.lookup(HaRemoteNamingStore.java:272)
	at org.jboss.naming.remote.client.RemoteContext.lookupInternal(RemoteContext.java:104)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:93)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:146)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at org.jboss.as.test.integration.ejb.mdb.containerstart.SendMessagesTestCase.testShutdown(SendMessagesTestCase.java:205)
Caused by: java.io.EOFException: Read past end of file
	at org.jboss.marshalling.SimpleDataInput.eofOnRead(SimpleDataInput.java:151)
	at org.jboss.marshalling.SimpleDataInput.readUnsignedByteDirect(SimpleDataInput.java:294)
	at org.jboss.marshalling.SimpleDataInput.readIntDirect(SimpleDataInput.java:343)
	at org.jboss.marshalling.SimpleDataInput.readLongDirect(SimpleDataInput.java:325)
	at org.jboss.marshalling.SimpleDataInput.readLong(SimpleDataInput.java:320)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadClassDescriptor(RiverUnmarshaller.java:1000)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadNewObject(RiverUnmarshaller.java:1255)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:276)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:209)
	at org.jboss.marshalling.AbstractObjectInput.readObject(AbstractObjectInput.java:41)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:156)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.BaseProtocolCommand.readResult(BaseProtocolCommand.java:59)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.handleClientMessage(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1$MessageReceiver$1.run(RemoteNamingStoreV1.java:232)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
------- Stdout: -------
15:49:08,723 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-6) WFLYSRV0028: Stopped deployment ContainerManagedTransactionNotSupportedTestCase.jar (runtime-name: ContainerManagedTransactionNotSupportedTestCase.jar) in 18ms
15:49:08,736 INFO  [org.jboss.as.repository] (management-handler-thread - 3) WFLYDR0002: Content removed from location C:\BuildAgent\work\a31d203e70e89f90\testsuite\integration\basic\target\jbossas\standalone\data\content\f2\fadc134b1cd18564abb18754ad5f6421a07a51\content
15:49:08,736 INFO  [org.jboss.as.server] (management-handler-thread - 3) WFLYSRV0009: Undeployed "ContainerManagedTransactionNotSupportedTestCase.jar" (runtime-name: "ContainerManagedTransactionNotSupportedTestCase.jar")
15:49:08,749 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "remove",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "ContainerManagedTransactionNotSupportedTestCaseQueueWithAnnotation")
    ]
}
15:49:08,762 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "remove",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "ContainerManagedTransactionNotSupportedTestCaseQueueWithDD")
    ]
}


org.jboss.as.test.integration.messaging.mgmt.ExportImportJournalTestCase.testExportImportJournal: org.jboss.naming.remote.protocol.NamingIOException: Failed to lookup
	at org.jboss.naming.remote.client.ClientUtil.namingException(ClientUtil.java:49)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.execute(Protocol.java:104)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1.lookup(RemoteNamingStoreV1.java:95)
	at org.jboss.naming.remote.client.HaRemoteNamingStore$1.operation(HaRemoteNamingStore.java:276)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.namingOperation(HaRemoteNamingStore.java:137)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.lookup(HaRemoteNamingStore.java:272)
	at org.jboss.naming.remote.client.RemoteContext.lookupInternal(RemoteContext.java:104)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:93)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:146)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at org.jboss.as.test.integration.messaging.mgmt.ExportImportJournalTestCase.sendMessage(ExportImportJournalTestCase.java:77)
	at org.jboss.as.test.integration.messaging.mgmt.ExportImportJournalTestCase.testExportImportJournal(ExportImportJournalTestCase.java:136)
Caused by: java.io.EOFException: Read past end of file
	at org.jboss.marshalling.SimpleDataInput.eofOnRead(SimpleDataInput.java:151)
	at org.jboss.marshalling.SimpleDataInput.readUnsignedByteDirect(SimpleDataInput.java:294)
	at org.jboss.marshalling.SimpleDataInput.readIntDirect(SimpleDataInput.java:343)
	at org.jboss.marshalling.SimpleDataInput.readLongDirect(SimpleDataInput.java:325)
	at org.jboss.marshalling.SimpleDataInput.readLong(SimpleDataInput.java:320)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadClassDescriptor(RiverUnmarshaller.java:1000)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadNewObject(RiverUnmarshaller.java:1255)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:276)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:209)
	at org.jboss.marshalling.AbstractObjectInput.readObject(AbstractObjectInput.java:41)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:156)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.BaseProtocolCommand.readResult(BaseProtocolCommand.java:59)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.handleClientMessage(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1$MessageReceiver$1.run(RemoteNamingStoreV1.java:232)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
------- Stdout: -------
15:50:12,273 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 72) AMQ221003: trying to deploy queue jms.queue.ExportImportJournalTestCase-Queue
15:50:12,283 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "ExportImportJournalTestCase-Queue")
    ],
    "entries" => ["java:jboss/exported/jms/ExportImportJournalTestCase-Queue"]
}
15:50:12,298 ERROR [org.jboss.as.naming] (pool-1-thread-9) WFLYNAM0008: Unexpected internal error: java.lang.NoClassDefFoundError: com/google/common/net/InetAddresses
	at org.apache.activemq.artemis.utils.IPV6Util.encloseHost(IPV6Util.java:39)
	at org.apache.activemq.artemis.uri.TCPServerLocatorSchema.escapeIPv6Host(TCPServerLocatorSchema.java:82)
	at org.apache.activemq.artemis.uri.TCPServerLocatorSchema.getURI(TCPServerLocatorSchema.java:75)
	at org.apache.activemq.artemis.uri.TCPSchema.internalNewURI(TCPSchema.java:66)
	at org.apache.activemq.artemis.uri.TCPSchema.internalNewURI(TCPSchema.java:31)
	at org.apache.activemq.artemis.utils.uri.URISchema.newURI(URISchema.java:45)
	at org.apache.activemq.artemis.utils.uri.URIFactory.createSchema(URIFactory.java:78)


org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.testChangeMessagePriority: org.jboss.naming.remote.protocol.NamingIOException: Failed to lookup
	at org.jboss.naming.remote.client.ClientUtil.namingException(ClientUtil.java:49)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.execute(Protocol.java:104)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1.lookup(RemoteNamingStoreV1.java:95)
	at org.jboss.naming.remote.client.HaRemoteNamingStore$1.operation(HaRemoteNamingStore.java:276)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.namingOperation(HaRemoteNamingStore.java:137)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.lookup(HaRemoteNamingStore.java:272)
	at org.jboss.naming.remote.client.RemoteContext.lookupInternal(RemoteContext.java:104)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:93)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:146)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.addQueues(JMSQueueManagementTestCase.java:95)
Caused by: java.io.EOFException: Read past end of file
	at org.jboss.marshalling.SimpleDataInput.eofOnRead(SimpleDataInput.java:151)
	at org.jboss.marshalling.SimpleDataInput.readUnsignedByteDirect(SimpleDataInput.java:294)
	at org.jboss.marshalling.SimpleDataInput.readIntDirect(SimpleDataInput.java:343)
	at org.jboss.marshalling.SimpleDataInput.readLongDirect(SimpleDataInput.java:325)
	at org.jboss.marshalling.SimpleDataInput.readLong(SimpleDataInput.java:320)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadClassDescriptor(RiverUnmarshaller.java:1000)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadNewObject(RiverUnmarshaller.java:1255)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:276)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:209)
	at org.jboss.marshalling.AbstractObjectInput.readObject(AbstractObjectInput.java:41)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:156)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.BaseProtocolCommand.readResult(BaseProtocolCommand.java:59)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.handleClientMessage(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1$MessageReceiver$1.run(RemoteNamingStoreV1.java:232)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
------- Stdout: -------
15:50:12,334 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 72) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCase1449499812331
15:50:12,344 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCase1449499812331")
    ],
    "entries" => ["java:jboss/exported/queue/JMSQueueManagementTestCase1449499812331"]
}
15:50:12,345 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 73) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCaseother1449499812331
15:50:12,356 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCaseother1449499812331")
    ],
    "entries" => ["java:jboss/exported/queue/JMSQueueManagementTestCaseother1449499812331"]


org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.testListAndCountMessages: org.jboss.naming.remote.protocol.NamingIOException: Failed to lookup
	at org.jboss.naming.remote.client.ClientUtil.namingException(ClientUtil.java:49)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.execute(Protocol.java:104)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1.lookup(RemoteNamingStoreV1.java:95)
	at org.jboss.naming.remote.client.HaRemoteNamingStore$1.operation(HaRemoteNamingStore.java:276)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.namingOperation(HaRemoteNamingStore.java:137)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.lookup(HaRemoteNamingStore.java:272)
	at org.jboss.naming.remote.client.RemoteContext.lookupInternal(RemoteContext.java:104)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:93)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:146)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.addQueues(JMSQueueManagementTestCase.java:95)
Caused by: java.io.IOException: Internal server error.
	at org.jboss.naming.remote.protocol.v1.RemoteNamingServerV1$MessageReciever$1.run(RemoteNamingServerV1.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
------- Stdout: -------
15:50:12,391 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 72) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCase1449499812332
15:50:12,401 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCase1449499812332")
    ],
    "entries" => ["java:jboss/exported/queue/JMSQueueManagementTestCase1449499812332"]
}
15:50:12,403 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 73) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCaseother1449499812332
15:50:12,412 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCaseother1449499812332")
    ],
    "entries" => ["java:jboss/exported/queue/JMSQueueManagementTestCaseother1449499812332"]
}
15:50:12,425 ERROR [org.jboss.as.naming] (pool-1-thread-3) WFLYNAM0008: Unexpected internal error: java.lang.NoClassDefFoundError: com/google/common/net/InetAddresses
	at org.apache.activemq.artemis.utils.IPV6Util.encloseHost(IPV6Util.java:39)
	at org.apache.activemq.artemis.uri.TCPServerLocatorSchema.escapeIPv6Host(TCPServerLocatorSchema.java:82)
	at org.apache.activemq.artemis.uri.TCPServerLocatorSchema.getURI(TCPServerLocatorSchema.java:75)
	at org.apache.activemq.artemis.uri.TCPSchema.internalNewURI(TCPSchema.java:66)
	at org.apache.activemq.artemis.uri.TCPSchema.internalNewURI(TCPSchema.java:31)
	at org.apache.activemq.artemis.utils.uri.URISchema.newURI(URISchema.java:45)
	at org.apache.activemq.artemis.utils.uri.URIFactory.createSchema(URIFactory.java:78)
	at org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory.toURI(ActiveMQConnectionFactory.java:119)
	at org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory.writeExternal(ActiveMQConnectionFactory.java:83)
	at org.jboss.marshalling.river.RiverMarshaller.doWriteObject(RiverMarshaller.java:842)
	at org.jboss.marshalling.AbstractObjectOutput.writeObject(AbstractObjectOutput.java:58)
	at org.jboss.marshalling.AbstractMarshaller.writeObject(AbstractMarshaller.java:111)


org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.testAddJndi: org.jboss.naming.remote.protocol.NamingIOException: Failed to lookup
	at org.jboss.naming.remote.client.ClientUtil.namingException(ClientUtil.java:49)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.execute(Protocol.java:104)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1.lookup(RemoteNamingStoreV1.java:95)
	at org.jboss.naming.remote.client.HaRemoteNamingStore$1.operation(HaRemoteNamingStore.java:276)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.namingOperation(HaRemoteNamingStore.java:137)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.lookup(HaRemoteNamingStore.java:272)
	at org.jboss.naming.remote.client.RemoteContext.lookupInternal(RemoteContext.java:104)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:93)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:146)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.addQueues(JMSQueueManagementTestCase.java:95)
Caused by: java.io.EOFException: Read past end of file
	at org.jboss.marshalling.SimpleDataInput.eofOnRead(SimpleDataInput.java:151)
	at org.jboss.marshalling.SimpleDataInput.readUnsignedByteDirect(SimpleDataInput.java:294)
	at org.jboss.marshalling.SimpleDataInput.readIntDirect(SimpleDataInput.java:343)
	at org.jboss.marshalling.SimpleDataInput.readLongDirect(SimpleDataInput.java:325)
	at org.jboss.marshalling.SimpleDataInput.readLong(SimpleDataInput.java:320)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadClassDescriptor(RiverUnmarshaller.java:1000)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadNewObject(RiverUnmarshaller.java:1255)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:276)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:209)
	at org.jboss.marshalling.AbstractObjectInput.readObject(AbstractObjectInput.java:41)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:156)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.BaseProtocolCommand.readResult(BaseProtocolCommand.java:59)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.handleClientMessage(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1$MessageReceiver$1.run(RemoteNamingStoreV1.java:232)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
------- Stdout: -------
15:50:12,445 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 72) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCase1449499812333
15:50:12,455 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCase1449499812333")
    ],
    "entries" => ["java:jboss/exported/queue/JMSQueueManagementTestCase1449499812333"]
}
15:50:12,457 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 73) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCaseother1449499812333
15:50:12,467 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCaseother1449499812333")
    ],
    "entries" => ["java:jboss/exported/queue/JMSQueueManagementTestCaseother1449499812333"]


org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.testRemoveLastJndi: org.jboss.naming.remote.protocol.NamingIOException: Failed to lookup
	at org.jboss.naming.remote.client.ClientUtil.namingException(ClientUtil.java:49)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.execute(Protocol.java:104)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1.lookup(RemoteNamingStoreV1.java:95)
	at org.jboss.naming.remote.client.HaRemoteNamingStore$1.operation(HaRemoteNamingStore.java:276)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.namingOperation(HaRemoteNamingStore.java:137)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.lookup(HaRemoteNamingStore.java:272)
	at org.jboss.naming.remote.client.RemoteContext.lookupInternal(RemoteContext.java:104)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:93)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:146)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.addQueues(JMSQueueManagementTestCase.java:95)
Caused by: java.io.EOFException: Read past end of file
	at org.jboss.marshalling.SimpleDataInput.eofOnRead(SimpleDataInput.java:151)
	at org.jboss.marshalling.SimpleDataInput.readUnsignedByteDirect(SimpleDataInput.java:294)
	at org.jboss.marshalling.SimpleDataInput.readIntDirect(SimpleDataInput.java:343)
	at org.jboss.marshalling.SimpleDataInput.readLongDirect(SimpleDataInput.java:325)
	at org.jboss.marshalling.SimpleDataInput.readLong(SimpleDataInput.java:320)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadClassDescriptor(RiverUnmarshaller.java:1000)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadNewObject(RiverUnmarshaller.java:1255)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:276)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:209)
	at org.jboss.marshalling.AbstractObjectInput.readObject(AbstractObjectInput.java:41)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:156)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.BaseProtocolCommand.readResult(BaseProtocolCommand.java:59)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.handleClientMessage(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1$MessageReceiver$1.run(RemoteNamingStoreV1.java:232)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
------- Stdout: -------
ead.run(Thread.java:745)

15:50:12,496 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 72) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCase1449499812334
15:50:12,505 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCase1449499812334")
    ],
    "entries" => ["java:jboss/exported/queue/JMSQueueManagementTestCase1449499812334"]
}
15:50:12,507 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 73) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCaseother1449499812334
15:50:12,517 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCaseother1449499812334")


org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.testMessageCounters: org.jboss.naming.remote.protocol.NamingIOException: Failed to lookup
	at org.jboss.naming.remote.client.ClientUtil.namingException(ClientUtil.java:49)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.execute(Protocol.java:104)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1.lookup(RemoteNamingStoreV1.java:95)
	at org.jboss.naming.remote.client.HaRemoteNamingStore$1.operation(HaRemoteNamingStore.java:276)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.namingOperation(HaRemoteNamingStore.java:137)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.lookup(HaRemoteNamingStore.java:272)
	at org.jboss.naming.remote.client.RemoteContext.lookupInternal(RemoteContext.java:104)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:93)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:146)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.addQueues(JMSQueueManagementTestCase.java:95)
Caused by: java.io.EOFException: Read past end of file
	at org.jboss.marshalling.SimpleDataInput.eofOnRead(SimpleDataInput.java:151)
	at org.jboss.marshalling.SimpleDataInput.readUnsignedByteDirect(SimpleDataInput.java:294)
	at org.jboss.marshalling.SimpleDataInput.readIntDirect(SimpleDataInput.java:343)
	at org.jboss.marshalling.SimpleDataInput.readLongDirect(SimpleDataInput.java:325)
	at org.jboss.marshalling.SimpleDataInput.readLong(SimpleDataInput.java:320)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadClassDescriptor(RiverUnmarshaller.java:1000)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadNewObject(RiverUnmarshaller.java:1255)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:276)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:209)
	at org.jboss.marshalling.AbstractObjectInput.readObject(AbstractObjectInput.java:41)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:156)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.BaseProtocolCommand.readResult(BaseProtocolCommand.java:59)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.handleClientMessage(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1$MessageReceiver$1.run(RemoteNamingStoreV1.java:232)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
------- Stdout: -------
15:50:12,546 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 72) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCase1449499812335
15:50:12,555 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCase1449499812335")
    ],
    "entries" => ["java:jboss/exported/queue/JMSQueueManagementTestCase1449499812335"]
}
15:50:12,557 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 73) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCaseother1449499812335
15:50:12,567 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCaseother1449499812335")
    ],
    "entries" => ["java:jboss/exported/queue/JMSQueueManagementTestCaseother1449499812335"]


org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.testRemoveJndi: org.jboss.naming.remote.protocol.NamingIOException: Failed to lookup
	at org.jboss.naming.remote.client.ClientUtil.namingException(ClientUtil.java:49)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.execute(Protocol.java:104)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1.lookup(RemoteNamingStoreV1.java:95)
	at org.jboss.naming.remote.client.HaRemoteNamingStore$1.operation(HaRemoteNamingStore.java:276)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.namingOperation(HaRemoteNamingStore.java:137)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.lookup(HaRemoteNamingStore.java:272)
	at org.jboss.naming.remote.client.RemoteContext.lookupInternal(RemoteContext.java:104)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:93)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:146)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.addQueues(JMSQueueManagementTestCase.java:95)
Caused by: java.io.EOFException: Read past end of file
	at org.jboss.marshalling.SimpleDataInput.eofOnRead(SimpleDataInput.java:151)
	at org.jboss.marshalling.SimpleDataInput.readUnsignedByteDirect(SimpleDataInput.java:294)
	at org.jboss.marshalling.SimpleDataInput.readIntDirect(SimpleDataInput.java:343)
	at org.jboss.marshalling.SimpleDataInput.readLongDirect(SimpleDataInput.java:325)
	at org.jboss.marshalling.SimpleDataInput.readLong(SimpleDataInput.java:320)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadClassDescriptor(RiverUnmarshaller.java:1000)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadNewObject(RiverUnmarshaller.java:1255)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:276)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:209)
	at org.jboss.marshalling.AbstractObjectInput.readObject(AbstractObjectInput.java:41)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:156)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.BaseProtocolCommand.readResult(BaseProtocolCommand.java:59)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.handleClientMessage(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1$MessageReceiver$1.run(RemoteNamingStoreV1.java:232)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
------- Stdout: -------
15:50:12,611 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 72) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCase1449499812336
15:50:12,621 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCase1449499812336")
    ],
    "entries" => ["java:jboss/exported/queue/JMSQueueManagementTestCase1449499812336"]
}
15:50:12,623 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 73) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCaseother1449499812336
15:50:12,633 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCaseother1449499812336")
    ],
    "entries" => ["java:jboss/exported/queue/JMSQueueManagementTestCaseother1449499812336"]


org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.testMessageMovement: org.jboss.naming.remote.protocol.NamingIOException: Failed to lookup
	at org.jboss.naming.remote.client.ClientUtil.namingException(ClientUtil.java:49)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.execute(Protocol.java:104)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1.lookup(RemoteNamingStoreV1.java:95)
	at org.jboss.naming.remote.client.HaRemoteNamingStore$1.operation(HaRemoteNamingStore.java:276)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.namingOperation(HaRemoteNamingStore.java:137)
	at org.jboss.naming.remote.client.HaRemoteNamingStore.lookup(HaRemoteNamingStore.java:272)
	at org.jboss.naming.remote.client.RemoteContext.lookupInternal(RemoteContext.java:104)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:93)
	at org.jboss.naming.remote.client.RemoteContext.lookup(RemoteContext.java:146)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at org.jboss.as.test.integration.messaging.mgmt.JMSQueueManagementTestCase.addQueues(JMSQueueManagementTestCase.java:95)
Caused by: java.io.EOFException: Read past end of file
	at org.jboss.marshalling.SimpleDataInput.eofOnRead(SimpleDataInput.java:151)
	at org.jboss.marshalling.SimpleDataInput.readUnsignedByteDirect(SimpleDataInput.java:294)
	at org.jboss.marshalling.SimpleDataInput.readIntDirect(SimpleDataInput.java:343)
	at org.jboss.marshalling.SimpleDataInput.readLongDirect(SimpleDataInput.java:325)
	at org.jboss.marshalling.SimpleDataInput.readLong(SimpleDataInput.java:320)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadClassDescriptor(RiverUnmarshaller.java:1000)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadNewObject(RiverUnmarshaller.java:1255)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:276)
	at org.jboss.marshalling.river.RiverUnmarshaller.doReadObject(RiverUnmarshaller.java:209)
	at org.jboss.marshalling.AbstractObjectInput.readObject(AbstractObjectInput.java:41)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:156)
	at org.jboss.naming.remote.protocol.v1.Protocol$1$3.read(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.BaseProtocolCommand.readResult(BaseProtocolCommand.java:59)
	at org.jboss.naming.remote.protocol.v1.Protocol$1.handleClientMessage(Protocol.java:149)
	at org.jboss.naming.remote.protocol.v1.RemoteNamingStoreV1$MessageReceiver$1.run(RemoteNamingStoreV1.java:232)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
------- Stdout: -------
15:50:12,716 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 72) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCase1449499812337
15:50:12,726 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCase1449499812337")
    ],
    "entries" => ["java:jboss/exported/queue/JMSQueueManagementTestCase1449499812337"]
}
15:50:12,728 INFO  [org.apache.activemq.artemis.core.server] (ServerService Thread Pool -- 73) AMQ221003: trying to deploy queue jms.queue.JMSQueueManagementTestCaseother1449499812337
15:50:12,739 INFO  [org.jboss.as.test.integration.common.jms.JMSOperationsProvider] (main) Operation successful for update = {
    "operation" => "add",
    "address" => [
        ("subsystem" => "messaging-activemq"),
        ("server" => "default"),
        ("jms-queue" => "JMSQueueManagementTestCaseother1449499812337")
    ],
    "entries" => ["java:jboss/exported/queue/JMSQueueManagementTestCaseother1449499812337"]



##### there are 19 more failed tests, see build details
```

--- a/feature-pack/src/main/resources/modules/system/layers/base/org/apache/activemq/artemis/main/module.xml
+++ b/feature-pack/src/main/resources/modules/system/layers/base/org/apache/activemq/artemis/main/module.xml
@@ -39,6 +39,8 @@
     </resources>
 
     <dependencies>
+        <!-- required for ARTEMIS-298 -->
+        <module name="com.google.guava"/>
         <module name="javax.api"/>
         <module name="javax.jms.api" />
         <module name="org.apache.commons.beanutils" />

--- a/pom.xml
+++ b/pom.xml
@@ -115,7 +115,7 @@
         <version.jsoup>1.8.3</version.jsoup>
         <version.net.jcip>1.0</version.net.jcip>
         <version.net.shibboleth.utilities.java-support>7.1.1</version.net.shibboleth.utilities.java-support>
-        <version.org.apache.activemq.artemis>1.1.0.wildfly-008</version.org.apache.activemq.artemis>
+        <version.org.apache.activemq.artemis>1.1.0.wildfly-009</version.org.apache.activemq.artemis>
         <version.org.apache.avro>1.7.6</version.org.apache.avro>
         <version.org.apache.cxf>3.1.4</version.org.apache.cxf>
         <version.org.apache.cxf.xjcplugins>3.0.5</version.org.apache.cxf.xjcplugins>

